excel_import_export/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     'name': 'Excel Import/Export',
     'summary': 'Base module for easy way to develop Excel import/export',
-    'version': '12.0.1.0.1',
+    'version': '12.0.1.0.2',
```

Comment:
FYI, merge bot does this, for avoid possible conflicts, so no need to do it yourself

Reply:
So, when it get merged manually, the version is not updated (I will update to .2 manually, hoping the bot will update to .3).
Just FYI.